doc/classes/SpatialMaterial.xml

```diff
@@ -157,10 +157,17 @@
 			Specifies whether to use [code]UV[/code] or [code]UV2[/code] for the detail layer. See [enum DetailUV] for options.
 		</member>
 		<member name="distance_fade_max_distance" type="float" setter="set_distance_fade_max_distance" getter="get_distance_fade_max_distance">
-			Distance at which the object fades fully and is no longer visible.
+            Distance at which the object appears at normal opaqueness.
+            
+            [b]Note:[/b] If max_distance is less than min_distance, then the behavior will be defined as:
+                Distance at which the object starts to fade. If the object is less than this distance away it will appear normal.
+               
 		</member>
 		<member name="distance_fade_min_distance" type="float" setter="set_distance_fade_min_distance" getter="get_distance_fade_min_distance">
-			Distance at which the object starts to fade. If the object is less than this distance away it will appear normal.
+            Distance at which the object starts to become less transparent.  If the object is less than this distance away, it will be invisible.
+            
+            [b]Note:[/b] If min_distance is greater than max_distance, then the behavior will be defined as:
```

Comment:
change to "[b]Note:[/b] If [code]distance_fade_min_distance[/code] is greater than [code]distance_fade_max_distance[/code], the behavior will be reversed, the object will start to fade away at [code]distance_fade_max_distance[/code] and fully disappear once it reaches [code]distance_fade_min_distance[/code].